core/src/main/java/org/apache/calcite/rel/mutable/MutableRels.java

```diff
@@ -173,6 +173,23 @@ public RexNode get(int index) {
         .collect(Collectors.toList());
   }
 
+  /**
+   * Construct expression list of Project by the given fields of the input.
+   */
+  public static List<RexNode> createProjects(final MutableRel child,
+      final List<RexNode> projs) {
```

Comment:
What does it do?
Is it really a public API?

What if I pass `+($0, 1)` as a project expression what would it do?